src/main/resources/META-INF/plugin.xml

```diff
@@ -63,6 +63,13 @@
                     text="Slide Out Current Branch"
                     description="Slide out current branch"
                     icon="AllIcons.Actions.GC"/>
+
+            <action id="GitMachete.PushAction"
+                    class="com.virtuslab.gitmachete.frontend.actions.PushAction"
+                    text="Push..."
```

Comment:
`Push Current Branch` for consistency/clarity? + add `description`

Reply:
I named it this way in the beginning but taking into account that we may push to multiple repositories that name would not be accurate. 

To be precise I decided to name it `Push...` for consistency. Let's take some other actions under the magnifying glass. `Commit...`, `Rollback...`, `Rename...`, `New Branch from Selected...`. From all of them, only the `Rollback...` has a description. The `...` denotes that this action requires something more, some specification or confirmation must be performed. For action like `Push...` this is particularly important.

Reply:
Things changes. Actually the suggested current/selected distinguishment has been introduced. Please rereview